src/style.css

```diff
@@ -0,0 +1,139 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --acdc-color: #00acdc;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
move to html and 
![image](https://user-images.githubusercontent.com/99331793/197973000-f5ec530a-7c54-4773-a5a6-a74c98b5a827.png)

Reply:
Which one is better to use as an alternative font for Roboto?

Reply:
Arial